arangod/Pregel/PregelFeature.h

```diff
@@ -90,6 +88,7 @@ class PregelFeature final : public application_features::ApplicationFeature {
  private:
   Mutex _mutex;
   std::unique_ptr<RecoveryManager> _recoveryManager;
+  std::atomic<RecoveryManager*> _recoveryManagerPtr;
```

Comment:
I suggest to add a comment here to explain that `_recoveryManagerPtr` always points to the same as `_recoveryManager` and why we need a separate, atomic pointer here. For example, it was not clear to me on first reading.

Reply:
Added a comment.